hr_holidays_public/models/resource_calendar.py

```diff
@@ -0,0 +1,77 @@
+# Copyright 2017-2018 Tecnativa - Pedro M. Baeza
```

Comment:
Entirely new file? Very wrong cherry-pick?

Reply:
Oops, the file name changed. Let's see now. It is my first time doing this.